openedx_events/data.py

```diff
@@ -27,6 +27,19 @@ def _ensure_utc_time(_, attribute, value):
     raise ValueError(f"'{attribute.name}' must have timezone.utc")
 
 
+def get_source_name():
```

Comment:
1. Should be `get_service_name`.
2. Can we keep this internal-only and call it `_get_service_name`?

Reply:
Also note that if this function were private, and just used for this one purpose, it could return `SERVICE-NAME-UNSET` itself in place of None.

Reply:
I wanted to call it from event bus kafka

Reply:
It should still be changed to `get_service_name` though, right?

Reply:
It was. I have 2 methods now